src/renderer/Cargo.toml

```diff
@@ -4,6 +4,7 @@ version = "0.3.1"
 authors = ["Eyal Kalderon <ebkalderon@gmail.com>"]
 description = "High-level rendering engine with multiple backends"
 keywords = ["game", "engine", "renderer", "3d", "amethyst"]
+categories = ["rendering"]
```

Comment:
Not entirely sure, rendering engine kind of brings to mind `gfx-rs` and such to me.